sections/graphrecs/graphrecs-chapter.tex

```diff
@@ -61,4 +61,13 @@ \section{Graph recommender systems}\label{sec:graph-rec-sys}
 The relevance between a user \textit{u} and an item \textit{i} can be evaluated using the proximity of \textit{u} to the item \textit{i}, meaning that the items that are "closest" in the graph to \textit{u} will be recommended.
 A similarity between a pair of users or items can also be found which can be expressed as a weight between them\cite{RecommenderHandbook2015}.
 
+\subsection{Side-information in graphs}
+With the structure of a graph, it is possible to extend it with side-information.
+This can be done either with information about the interaction between user and an item by adding side-information to the edge between the two nodes, such that the edge $(U, I)$ that previously indicated an interaction between an user and an item is extended with edge side-information such as $(U, r, I)$ where $r$ is a rating that user $U$ has assigned to item $I$\cite{aggarwal2016recommender}. 
```

Comment:
```suggestion
This can be done either with information about the interaction between user and an item by adding side-information to the edge between the two nodes, such that the edge $(u, i)$ that previously indicated an interaction between a user and an item is extended with side-information such as $(u, r, i)$, where $r$ is a rating that user $u$ has assigned to item $i$\cite{aggarwal2016recommender}. 
```